api-operator/pkg/controller/api/api_controller.go

```diff
@@ -670,6 +672,18 @@ func (r *ReconcileAPI) Reconcile(request reconcile.Request) (reconcile.Result, e
 		r.recorder.Event(instance, corev1.EventTypeNormal, "Deploy",
 			fmt.Sprintf("Successfully deployed the API: %s.", instance.Name))
 
+		if importAPIEnabled {
+			importErr := apim.ImportAPI(&r.client, instance)
+			if importErr != nil {
+				r.recorder.Event(instance, eventTypeError, "Import",
+					fmt.Sprintf("Error occured while importing the API to APIM"))
+				return reconcile.Result{}, importErr
+			}
+			r.recorder.Event(instance, corev1.EventTypeNormal, "Import",
```

Comment:
Shall we rename reason for this event also, something like "APIMImported" or "ImportedToAPIM"

Reply:
Fixed in e0400b3